intro_to_R_tidyverse/intro_to_r.Rmd

```diff
@@ -0,0 +1,391 @@
+  
+This html is made from an [RMarkdown file](https://rmarkdown.rstudio.com/articles_intro.html). 
+Rmd's are handy for times you'd like to have output that has the code along with
+the output from that code. 
+Rmds also have a lot of options and features for what that output looks like and 
+what it includes. 
+
+## The Basics  
+R remembers what you tell it and stores assigned values in it's environment. 
+In RStudio, you will see what R has in it's environment in the "environment" panel.
+Here let's assign the value of `1` to `x` using the assignment operator: `<-`.
+```{r Basic assignment}
+x <- 1
+```
+
+Now R will print out `1` if we tell it `x`.
+```{r Recall}
+x
+```
+
+Use the print function to more explicitly ask R to recall what x is
+```{r print}
+print(x)
+```
+
+R will ignore what you say (allowing you to make comments on your code) if you 
+put hashtags/pound signs.
+```{r Comments}
+# This is important because you should always have comments to explain your code
+```
+
+Can't remember what values you have assigned to things in your environment? 
+Use the function `ls` to list everything in the current environment. 
+Just like in math, 
+*functions* are things that take input, transform or otherwise do something with this input, and then give you output. 
+In R, functions take input through *arguments* within their parantheses. 
+If the arguments don't make sense, you will get an *error* that will try to tell 
+you why R can't do what you asked. 
+Use a `?` mark before a function name to bring up the help page for that function
+so you can see how it's supposed to be used. 
+Or just search for the function by name in the *help bar* in RStudio.
+```{r Environment}
+# It will tell us that our object "x" is in the environment
+ls()
+
+# Let's look at the info for the ls() function:
+?ls
+```
+
+Based on the info we saw when we ran `ls()`, what would happen if we got rid of the
+pound sign and tried to run this following line?:
+```{r }
+# print(y)
+```
+
+Base R (R without any extra frills from *packages*) knows how to do arithmetic. 
+Feel free to try out division, exponents, and multiplication.   
+*Hint:* `/`  `^`  `*` 
+```{r Arithmetic}
+# Addition:
+x + 5
+```
+
+## Data Types  
+R classifies objects as different data types. 
+We'll go through examples of each so it's clear what these are; here's a brief
+explanation of each:  
+  - *Numeric* : Numbers like 1.3333 or 1  
+  - *Integer* : Numbers without decimals   
+  - *Character* : Things with quotes around them; strings of characters  
+  - *Factor* : Sets of variables that R will assign an factor levels to  
+  - *Logical* : An object where 0 means FALSE and 1 means TRUE  
+
+Based on these categories, what kind of data structure do you think `x` is right
+now? 
+If we want to find out about the object `x` data type and structure, we can use 
+the function `str` to find out. 
+```{r Data structures}
+# Ask R to tell you what type of data structure "x" is
```

Comment:
Data Types and Data Structures are used interchangeably. Maybe refer to it as `Data Structures` throughout, if that doesn't conflict with another concept in R?

Reply:
Just got down to the `Data Structure` section. Perhaps make the distinction a little more apparent? I'm not sure but it's a little confusing.

Reply:
I've added some more up front definitions of *types* v *structures* hopefully that helps.